src/negotiation/execution_params.spec.ts

```diff
@@ -0,0 +1,89 @@
+import moment = require("moment");
+import { ExecutionParams, validateExecutionParams } from "./execution_params";
+
+describe("Taker Negotiator", () => {
+  it("Validates mainnet execution parameters", () => {
+    const executionParams: ExecutionParams = {
```

Comment:
Could extract an instance with `defaultExecutionParams` (just local to this file) and then use that with spread notation to override the what is relevant to this test:

```js
const executionParams: {
   ...defaultExecutionParams(),
   // override here
};
```

Reply:
Dang, if this syntax works then I need to use more often!